libraries/helpers.rb

```diff
@@ -53,7 +53,11 @@ def pip_command(command)
       def pyenv_prerequisites
         case node['platform_family']
         when 'debian'
-          %w(make libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git)
+          %w(make libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev git) << if platform?('ubuntu') && node['lsb']['codename'] == 'jammy'
```

Comment:
You shouldn’t be comparing the code name. This isn’t future proof and will be broken with next release of Ubuntu even if there is no change. You should be comparing the platform version. This also is specific to Ubuntu when typically Debian and Ubuntu share the same package names. Have you verified Debian’s latest version isn’t using python3-OpenSSL as well?

Reply:
thanks for the comment. Yeah, it makes more sense checking the version.

you are also right about debian - bullseye and on (last stable) seems to not have python-openssl as well (https://packages.debian.org/bullseye/python-openssl vs https://packages.debian.org/bullseye/python3-openssl)

python-openssl exists up to buster https://packages.debian.org/buster/python-openssl

I'll make the changes.